gfauto/gfauto/amber_converter.py

```diff
@@ -355,12 +363,17 @@ def amberscript_uniform_buffer_def(uniform_json_contents: str, prefix: str) -> s
             raise ValueError("Error: unknown uniform type for function: " + func)
         uniform_type = uniform_types[func]
```

Comment:
Please add an example to the above docstring for a sampler.

Reply:
Done